data/wanderer/wanderers start.txt

```diff
@@ -768,6 +769,14 @@ mission "Wanderers: Unfettered Diplomacy 1B"
 		personality timid escort
 		ship "Deep River (Jump, Unarmed)" "Ayi'i Ki'ak'var"
 
+fleet "Unfettered Tribute (Wanderer)"
+	government "Hai (Unfettered) (Wanderer Tribute)"
+	names "hai"
+	cargo 3
+	variant
+		"Water Bug (Jump)"
+
+# TODO: this fleet is considered deprecated, but preserved for old save files with references to it.
```

Comment:
```suggestion
```
the fleet was used at 3 places, I see no reason to not just change the government, this is exactly the kind of thing I'd like to avoid because it would make the eventual rework a mess

Reply:
Also these ships should 100% be made coward.